docs/pages/2019_MARVEL_Psik_MaX/sections/setup.rst

```diff
@@ -138,11 +140,27 @@ terminal is the one we will actually use in this tutorial.
 
 .. note::
 
-   Our SSH configuration assumes that ``jupyter`` will serve the notebooks on port 8888. 
+   Our SSH configuration assumes that ``jupyter`` will serve the notebooks on port 8888.
    If you want to serve notebooks on different ports, you'll also need to adjust
    the SSH configuration.
 
 
+.. _setup_downloading_files:
+
+Downloading files
+-----------------
+
+Throughout this tutorial, you might encounter links through which you can download files, example and or solution scripts for example.
+Often the idea is to click on the link to download the file and put it in your working directory for the tutorial.
+If you are running the tutorial code on a different machine from which you are reading this tutorial, you can always directly download the files using ``wget``.
+Simply copy the address of the link and execute the following command:
+
+   wget <LINK>
+
+where you replace ``<LINK>`` with the actual HTTPS link that you copied from the tutorial text in your browser.
+This will download that file in your current directory.
```

Comment:
and just check once that this works (i.e. that filenames are correct when downloading from the web site. I think they will be)

Reply:
I tried with a link that is currently online and it worked, it will copy it to current dir with same filename